test/metabase/models/setting_test.clj

```diff
@@ -576,119 +578,8 @@
     (is (= "Banana Beak"
            (toucan-name)))))
 
-(deftest duplicated-setting-name
```

Comment:
These tests I just moved down to the bottom of the file under the "Misc Tests" heading